robot.py

```diff
@@ -211,76 +224,29 @@ def robotPeriodic(self):
                 * 10  # convert to seconds
                 / module.drive_counts_per_metre,
             )
+        self.sd.putBoolean("heading_hold", self.chassis.hold_heading)
```

Comment:
?

Reply:
To allow easy checks if the heading hold is on